### PR TITLE
eventpp: new recipe

### PR DESCRIPTION
**Pull request by RazielXYZ**
Specify library name and version:  **eventpp/0.1.3**

[Eventpp](https://github.com/wqking/eventpp) is a C++11 header-only library for thread-safe callbacks, event dispatchers, and event queues.

---

- [x] I've read the [contributing guidelines](https://github.com/conan-io/conan-center-index/blob/master/CONTRIBUTING.md).
- [x] I've used a [recent](https://github.com/conan-io/conan/releases/latest) Conan client version close to the [currently deployed](https://github.com/conan-io/conan-center-index/blob/master/.c3i/config_v1.yml#L6).
- [x] I've tried at least one configuration locally with the [conan-center hook](https://github.com/conan-io/hooks.git) activated.

### CODE REVIEW

File: recipes/eventpp/all/conanfile.py [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+    package_type = "header-library"
+    settings = "os", "arch", "compiler", "build_type"
+
+    @property
+    def _min_cppstd(self):
+        return 11
+
+    def export_sources(self):
+        export_conandata_patches(self)
+
```
Comment (RubenRBS) [inline recipes/eventpp/all/conanfile.py:26]: No need to call this if the recipe contains no patches :)
```suggestion
```

File: recipes/eventpp/all/conanfile.py [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+    def validate(self):
+        if self.settings.get_safe("compiler.cppstd"):
+            check_min_cppstd(self, self._min_cppstd)
+
+    def source(self):
+        get(self, **self.conan_data["sources"][self.version], strip_root=True)
+
+    def build(self):
+        apply_conandata_patches(self)
+
```
Comment (RubenRBS) [inline recipes/eventpp/all/conanfile.py:42]:
```suggestion
```

File: recipes/eventpp/all/test_v1_package/CMakeLists.txt
Comment (RubenRBS) [inline recipes/eventpp/all/test_v1_package/CMakeLists.txt:1]: No need for test_v1_packages for new recipes, but won't hurt if you leave it now that it's here :)

**Review by RubenRBS — CHANGES_REQUESTED** [review]

**Review by RubenRBS — APPROVED** [review]
Thanks a lot! The recipe looks great, I've just pushed a few minor changes, otherwise looks great :)

**Review by jwillikers — APPROVED** [review]

**Comment by wqking** [comment]
Hi, thanks for adding eventpp to Conan. After you finish, can you please create an issue in eventpp repo to give me a piece of instruction on how to use Conan so I can add it to the document?

**Pull request merged**